mappings/net/minecraft/sound/SoundCategory.mapping

```diff
@@ -1,5 +1,4 @@
 CLASS net/minecraft/class_3419 net/minecraft/sound/SoundCategory
-	FIELD field_15248 PLAYERS Lnet/minecraft/class_3419;
```

Comment:
?

Reply:
Inferred enum field, happened to encounter so i removed it